apps/condo/domains/user/schema/User.js

```diff
@@ -144,12 +144,17 @@ const User = new GQLListSchema('User', {
             // TODO(pahaz): we should check the structure!
         },
 
-        // TODO(pahaz): should we also add remote system?
+        importRemoteSystem: {
+            schemaDoc: 'External provider for users',
+            type: Text,
+            access: access.canAccessToImportField,
+            kmigratorOptions: { null: true, unique: false },
+        },
         importId: {
             schemaDoc: 'External system user id. Used for integrations',
             type: Text,
-            access: access.canAccessToImportIdField,
-            kmigratorOptions: { null: true, unique: true },
+            access: access.canAccessToImportField,
+            kmigratorOptions: { null: true, unique: false },
```

Comment:
Is it possible to have multiple different user with one integration?

Reply:
There is a constraint to make a combination of ImportId + ImportRemoteSystem to be unique